pkg/server/computeRecommendation.go

```diff
@@ -62,41 +67,51 @@ func computeRecommendations(w http.ResponseWriter, r *http.Request) {
 		}
 		return
 	}
-
```

Comment:
use klog in line 65 instead of log.
`log.Print(err.Error())`

Reply:
Done